gateway/ln-gateway/src/lib.rs

```diff
@@ -248,22 +248,19 @@ impl Gateway {
                             break;
                         }
 
-                        // Create a stream used to communicate with the Lightning implementation
-                        let (sender, ln_receiver) = mpsc::channel::<InterceptHtlcResponse>(100);
-
                         if let Some(ln_mode) = ln_mode.clone() {
-                            let mut lnrpc = Self::create_boxxed_lightning_client(ln_mode).await;
+                            let mut lnrpc_route = Self::create_boxxed_lightning_client(ln_mode).await;
```

Comment:
I know it's unrelated but could fix typo:
```suggestion
                            let mut lnrpc_route = Self::create_boxed_lightning_client(ln_mode).await;
```